src/main/java/no/nav/pto/veilarbportefolje/registrering/RegistreringRepositoryV2.java

```diff
@@ -0,0 +1,104 @@
+package no.nav.pto.veilarbportefolje.registrering;
+
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import no.nav.arbeid.soker.registrering.ArbeidssokerRegistrertEvent;
+import no.nav.arbeid.soker.registrering.UtdanningBestattSvar;
+import no.nav.arbeid.soker.registrering.UtdanningGodkjentSvar;
+import no.nav.arbeid.soker.registrering.UtdanningSvar;
+import no.nav.common.types.identer.AktorId;
+import no.nav.pto.veilarbportefolje.util.DateUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import java.sql.ResultSet;
+import java.sql.Timestamp;
+import java.util.Optional;
+
+import static java.util.Optional.ofNullable;
+import static java.time.Instant.now;
+import static no.nav.pto.veilarbportefolje.database.PostgresTable.BRUKER_REGISTRERING.*;
+import static no.nav.pto.veilarbportefolje.postgres.PostgresUtils.queryForObjectOrNull;
+
+import java.time.format.DateTimeFormatter;
+
+@Slf4j
+@Repository
+public class RegistreringRepositoryV2 {
+    private final JdbcTemplate db;
```

Comment:
Mer for gøy, sjekk om dette fungerer:  `@NonNull  @Qualifier("PostgresJdbc") private final JdbcTemplate db;`

Reply:
og bruk av \@RequierdArgs...